developersociety/django-template
# Wagtail start

Description
Add block definitions and updates styles appropriately

Setup instructions
cookiecutter django-template after pulling locally on this branch. Create a wagtail project and make reset

How to test
Add a page with all the available wagtail blocks and make sure it looks like a solid starting point for the blocks. Navigation and global styles will be amended soon
